### PR TITLE
Upland integration sms

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

**Review by schuyler1d — COMMENTED** [review]
What's the relationship to these things with actionkit?  Do they need to be connected or can they be sequential action_handlers?  maybe the order can pass data through multiples -- e.g. if we lookup first/last name on the first, we don't need to get it for the second.

File: src/server/action_handlers/revere-signup.js [hunk 1]
```diff
@@ -26,58 +27,6 @@ export async function available(organizationId) {
   return false
 }
 
-const actionKitSignup = (cell, contact) => {
```
[the remark this answers is not in the document]
Comment (shakalee14) [reply src/server/action_handlers/revere-signup.js:29]: per our convo in slack: im not removing revere, and just moving ak signup to another file. ak posting also will only happen if ak config is included in the environment variables.

File: src/server/action_handlers/mobilecommons-signup.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import request from 'request'
+import aws from 'aws-sdk'
+import { r } from '../models'
+import { actionKitSignup } from './helper-ak-sync.js'
+// What the user sees as the option
+export const displayName = () => 'Mobile Commons Signup'
+
+const akAddUserUrl = process.env.AK_ADD_USER_URL
```
[the remark this answers is not in the document]
Comment (shakalee14) [reply src/server/action_handlers/mobilecommons-signup.js:8]: AK posting happens after we subscribe them to upland. these folks are brand new subscribers so we dont have an ak record for them.

**Review by schuyler1d — APPROVED** [review]
:+1:

**Pull request merged**